network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_directory.html

```diff
@@ -1,18 +1,21 @@
-{% load static %}
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags static %}
 
-<div class="profile-directory">
+{% block block_content  %}
   <div class="fellowships-directory-filter px-2 mb-5">
     {% for label in filters %}
     <div class="filter-option">
-      <button {% if forloop.first %}class="active"{% endif %}>{{ label }}</button>
+      <button {% if forloop.first %}class="active" {% endif %}>{{ label }}</button>
     </div>
     {% endfor %}
   </div>
+  </div>
```

Comment:
This looks like a duplicated line, does it need to be here? If so, let's fix the indentation